### PR TITLE
Reserve FactorUpdates Late August

### PR DESCRIPTION
**Pull request by defijesus**

### CODE REVIEW

File: src/20240821_Multi_ReserveFactorUpdatesLateAugust/config.ts [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    date: '20240821',
+    author: 'karpatkey_TokenLogic',
+    discussion:
+      'https://governance.aave.com/t/arfc-increase-bridged-usdc-reserve-factor-across-all-deployments/17787',
+    snapshot:
+      'https://snapshot.org/#/aave.eth/proposal/0x9cc7906f04f45cebeaa48a05ed281f49da00d89c4dd988a968272fa179f14d06',
+    votingNetwork: 'POLYGON',
+  },
+  poolOptions: {
+    AaveV2Ethereum: {configs: {OTHERS: {}}, cache: {blockNumber: 20592360}},
```
Comment (pavelvm5) [inline src/20240821_Multi_ReserveFactorUpdatesLateAugust/config.ts:24]: There's no updates here, however RF is updated.

File: src/20240821_Multi_ReserveFactorUpdatesLateAugust/config.ts [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+              variableRateSlope2: '',
+              stableRateSlope1: '',
+              stableRateSlope2: '',
+            },
+          },
+        ],
+      },
+      cache: {blockNumber: 60947312},
+    },
+    AaveV2Avalanche: {configs: {OTHERS: {}}, cache: {blockNumber: 49543127}},
```
Comment (pavelvm5) [inline src/20240821_Multi_ReserveFactorUpdatesLateAugust/config.ts:98]: The same problem here.

**Comment by brotherlymite** [comment]
@defijesus does it make sense to increase the Reserve Factor for USDC on the gnosis chain as we had a [new USDC listing](https://vote.onaave.com/proposal/?proposalId=139&ipfsHash=0xf419f34373957ef660343043d18871e0d104ecc83709743658edeb1d44edd6f2) on gnosis instance.

**Review by pavelvm5 — DISMISSED** [review]

**Review by brotherlymite — APPROVED** [review]

File: src/20240821_Multi_ReserveFactorUpdatesLateAugust/ReserveFactorUpdatesLateAugust.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+| DAI    | Ethereum v2  |   70.00%   |   75.00%    |
+| LINK   | Ethereum v2  |   75.00%   |   80.00%    |
+| USDC   | Ethereum v2  |   70.00%   |   75.00%    |
+| USDT   | Ethereum v2  |   70.00%   |   75.00%    |
+| wBTC   | Ethereum v2  |   75.00%   |   80.00%    |
+| wETH   | Ethereum v2  |   70.00%   |   75.00%    |
+| USDC.e |   Arbitrum   |   35.00%   |   40.00%    |
+| USDC.e |   Optimism   |   35.00%   |   40.00%    |
+| USDC.e |   Polygon    |   35.00%   |   40.00%    |
+| USDbC  |     Base     |   35.00%   |   40.00%    |
```
Comment (pavelvm5) [inline src/20240821_Multi_ReserveFactorUpdatesLateAugust/ReserveFactorUpdatesLateAugust.md:58]: Could you add info about Gnosis market changes here too?

**Review by pavelvm5 — APPROVED** [review]

**Review by brotherlymite — APPROVED** [review]

**Pull request merged**